tests/console/nested_vm.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: nest_vm
+# Summary: Basics libvirtd test
+# Maintainer: wegao@suse.com
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use services::libvirtd;
+
+sub run {
+    select_console 'root-console';
+    if (get_var('NESTED_VM_CHECK_BEFORE_MIGRATION')) {
+        my %hash = (stage => 'after', service_type => 'Systemd', srv_pkg_name => 'libvirtd');
+        services::libvirtd::full_libvirtd_check(%hash);
+    }
+    else {
+        my %hash = (stage => 'before', service_type => 'Systemd', srv_pkg_name => 'libvirtd');
+        services::libvirtd::full_libvirtd_check(%hash);
+        set_var('NESTED_VM_CHECK_BEFORE_MIGRATION', 1);
+        enter_cmd "reboot";
+    }
+}
+
+sub post_run_hook {
+    # Do nothing
+}
```

Comment:
Remove these lines?

Reply:
This is used to override default post_run_hook:)